Description: Use upstream version
 We want to use the upstream versioning information from the autorevision cache.
Forwarded: not-needed
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2018-03-16
--- a/ntpd/wscript
+++ b/ntpd/wscript
@@ -19,7 +19,7 @@
 
         ctx(
             cwd=srcnode,
-            rule='VCS_EXTRA=`cat ${SRC[0]}` wafhelpers/autorevision.sh '
+            rule='VCS_EXTRA=`cat ${SRC[0]}` wafhelpers/autorevision.sh -f '
                  '-o ${TGT[1].abspath()} -e VERSION -t h >${TGT[0].abspath()}',
             source=["../VERSION", '../wafhelpers/autorevision.sh'],
             target=[target3, target4],
--- a/pylib/wscript
+++ b/pylib/wscript
@@ -52,7 +52,7 @@
     ctx(
         before=['pyc', 'pyo'],
         cwd=srcnode,
-        rule='VCS_EXTRA=`cat ${SRC[0]}` ../wafhelpers/autorevision.sh '
+        rule='VCS_EXTRA=`cat ${SRC[0]}` ../wafhelpers/autorevision.sh -f '
              '-o ${TGT[1].abspath()} -e VERSION -t python >${TGT[0].abspath()}',
         source=["../VERSION", '../wafhelpers/autorevision.sh'],
         target=[target3, target4],
